godtools/App/Features/Menu/Domain/UseCases/GetUserActivityBadgeUseCase/UserActivityBadgeDomainModel.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  UserActivityBadgeDomainModel.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 1/17/23.
+//  Copyright © 2023 Cru. All rights reserved.
+//
+
+import Foundation
+import GodToolsToolParser
+import SwiftUI
+
+struct UserActivityBadgeDomainModel {
+        
+    let badgeText: String
+    let badgeType: Badge.BadgeType
```

Comment:
I would say ```badgeType``` and ```variant``` shouldn't be exposed here since those attributes don't apply to the presentation layer.  Instead this struct should require the ```Identifiable``` ```id``` on initialization.  Something like ```identifiableId``` would be more clear.